scripts/globals/bcnm.lua

```diff
@@ -112,7 +112,7 @@ local battlefields = {
 
     [36] = {                -- EMPYREAL PARADOX
         { 0, 1056,    0},   -- Dawn (PM8-4)
-     -- { 1, 1057,    0},   -- Apocalypse Nigh (Quest)
+        { 1, 1057,    0},   -- Apocalypse Nigh (Quest)
```

Comment:
Don't turn this BCNM on until its content is fully coded (mobs, drops, etc.)

Reply:
Mobs are in there, no drops.